### PR TITLE
fix: disable life support for nearlib test

### PR DESCRIPTION
**Pull request by matklad**
What this test is intended to do is to spawn a local neard node and to run near-api-js tests against that.

The purpose is to detect unintentional changes to near public API.

We have disabled actually running the test a while ago, and today the build broke b/c api-js moved from yarn to pnpm.

I think at this it's safe to kill this infra completely, as we weren't running it for a while.

It would of course be nice to have some in-repo tests covering public JSON API, but I think we are currently server well-enough by betanet and testnet anyway.

Testing public APIs via running tests of downstream projects upstream is clearly a wrong way to do that!

### CODE REVIEW

**Comment by matklad** [comment]
@morgsmccauley  (random person from near-api-js), how is near-api-js is tested? Specifically, when running the tests in CI, do you also run the node, or is it fully mocked?

**Review by nikurt — APPROVED** [review]

**Comment by morgsmccauley** [comment]
@matklad it uses `https://rpc.ci-testnet.near.org` directly. I wasn't aware of these tests, sorry for the breaking change.

**Comment by matklad** [comment]
Thanks! 

So this means that, if we do break our APIs, we'll notice this at least when we hit testnet. This makes me confident that we are somewhat covered here.

File: scripts/nodelib.py [hunk 1]
```diff
@@ -254,15 +254,15 @@ def setup_and_run(nodocker,
...
-            exit(1)
+    # if nodocker:
+    #     install_cargo()
+    # else:
+    #     try:
+    #         subprocess.check_output(['docker', 'pull', image])
+    #         subprocess.check_output(['docker', 'pull', 'v2tec/watchtower'])
+    #     except subprocess.CalledProcessError as exc:
+    #         print("Failed to fetch docker containers: %s" % exc)
+    #         exit(1)
```
Comment (matklad) [inline scripts/nodelib.py:265]: Ouch, this definitely wasn't meant to be commited, sorry.

**Pull request merged**